<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerfProfiler::Internal::PerfLoadDialog</class>
 <widget class="QWidget" name="PerfProfiler::Internal::PerfLoadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Perf Trace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>&amp;Trace file:</string>
       </property>
       <property name="buddy">
        <cstring>traceFileLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="traceFileLineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="browseTraceFileButton">
       <property name="text">
        <string>&amp;Browse...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>Directory of &amp;executable:</string>
       </property>
       <property name="buddy">
        <cstring>executableDirLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="executableDirLineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="browseExecutableDirButton">
       <property name="text">
        <string>B&amp;rowse...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label3">
       <property name="text">
        <string>Kit:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="ProjectExplorer::KitChooser" name="kitChooser" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectExplorer::KitChooser</class>
   <extends>QWidget</extends>
   <header location="global">projectexplorer/kitchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
</ui>
